packages/aws-cdk/test/integ/run-against-dist.bash

```diff
@@ -4,8 +4,6 @@ npmws=/tmp/cdk-rundist
 rm -rf $npmws
 mkdir -p $npmws
 
-set -x
-
```

Comment:
intentional?

Reply:
Yes

Reply:
Twas there for debugging beforehand, I think